health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/appConfigurationScreen/DrawerFieldComposer.js

```
@@ -1,13 +1,94 @@
+          selected={state?.MASTER_DATA?.AppFieldType?.find((i) => i.type === drawerState?.type)}
+          select={(value) => {
+            setDrawerState((prev) => ({
+              ...prev,
+              type: value?.type,
+            }));
+          }}
+        />
+      );
+    default:
+      return null;
+      break;
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)